modules/@apostrophecms/page/index.js

```diff
@@ -2062,47 +2062,11 @@ database.`);
           await change(req, page, data);
         }
       },
-      // Given a page and its parent (if any), returns a schema that
-      // is filtered appropriately to that page's type, taking into
-      // account whether the page is new and the parent's allowed
-      // subpage types
+      // Backward compatible method following moving this to page-type module.
+      // This page module method may be deprecated in the next major version.
```

Comment:
OK

Reply:
(I was going to say keep it as a wrapper that takes care of finding the right manager, but you already have the manager for another reason in the one place you'd use it, so.)